lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+module Players
+  class Human < Player
+    def move(board)
+      puts "Please enter 1-9 for your move:"
+      index = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Please enter 1-9 for your move:"
  gets.strip
end
```